src/etc/inc/web/backup.inc

```diff
@@ -39,6 +39,12 @@ require_once("pkg-utils.inc");
 
 $rrddbpath = "/var/db/rrd";
 $rrdtool = "/usr/bin/nice -n20 /usr/local/bin/rrdtool";
+$backuppath = array(
+	'captiveportal' => "/var/db/captiveportal*.db",
+	'dhcpd' => "/var/dhcpd/var/db/dhcpd.leases",
+	'dhcpdv6' => "/var/dhcpd/var/db/dhcpd6.leases",
+	'voucher' => "/var/db/voucher_*.db"
+);
```

Comment:
Consider keeping these definitions in /etc/inc/globals.inc. That way they could be accessed via the $g[] global variable and there would be no need to keep track of them in multiple places. It would also avoid the need for another global in this function, which I would like to avoid wherever possible. Globals tend to break the separation of displa/logic.